crates/polars-arrow/src/compute/concatenate.rs

```diff
@@ -315,6 +318,7 @@ fn concatenate_view<V: ViewType + ?Sized, A: AsRef<dyn Array>>(
         Arc::new([Buffer::from(new_buffer)]) as Arc<[_]>
     };
 
+    dbg!(buffers.len());
```

Comment:
Stray dbg!

Reply:
I had already fixed it, forgot to push I think.